package.json

```diff
@@ -67,6 +67,7 @@
 		"fs-nextra": "^0.5.1",
 		"gifencoder": "^2.0.1",
 		"he": "^1.2.0",
+		"html-entities": "^1.3.1",
```

Comment:
We already have [`he`](https://www.npmjs.com/package/he) which removes HTML entities.

Reply:
> 
> 
> We already have [`he`](https://www.npmjs.com/package/he) which removes HTML entities.

Can we get `she` as an alias